README.md

```diff
@@ -1,2 +1,174 @@
+   <i> PCB Layout </i><br><br>
+  <img src="https://github.com/crooknish/Smartwatch/blob/328de0d6db4d20f19190899e9200ea6b46cae17e/Images%20and%20Videos/Images/PCB-Top%20view.png" height ="300px" width ="300px" alt="PCB top view">    <img src="https://github.com/crooknish/Smartwatch/blob/19d6efc3654f7e7757a4fd50ef805bb6503ef1a1/Images%20and%20Videos/Images/PCB%20-%20Bottom%20view.png" height ="300px" width ="300px" alt="PCB bottom view"><br>
+
+  </p>
+
+ 
+ 
+ ### BATTERY MANAGEMENT SYSTEM (BMS)
+ <p align="justify"> 
+  <b>Why use BMS:</b> Battery management system (BMS) are electronic control circuits that monitor and regulate the charging and discharge of batteries.
+The battery characteristics to be monitored include the detection of battery type, voltages, temperature, capacity, state of charge, power consumption, remaining operating time charging cycles, etc.  
+</p>
+
+<p align="justify"> 
+  <b>Task :</b> The task of BMS is to ensure the optimal use of the residual energy present in a battery, in order to avoid loading the batteries, BMS protect the batteries from deep discharge, from over-voltage, which are results to extreme fast charge and extreme high discharge current. In the case of multicell batteries, the battery management system also provides for cell balancing function, to manage that different battery cells have the same charging and discharging requirements.   
+  </p>
+ 
+
+<p align="justify">
+  <b>Charge protection IC’s :</b> For portable devices are offered featuring a withstand voltage up to 28V and multiple built-in protection circuits, including overvoltage lockout, over current limiter, undervoltage malfunction prevention, startup delay, and status flag, that provide the necessary protection for smart watch and other power consumer products. </p>
+
+#### Battery Charging System 
+<p align="justify"> One of the basic tasks in the battery charge is the regulation of battery voltage and current without exceeding the temperature limits. This requires a control loop that involves measurement of the battery parameters (like- voltages, current, and temperature) and controlling the PWM (Pulse Width Modulation) duty cycle that drives the external power network. PSOC (programmable system on a chip) with its precision ADC (Analog-to-Digital Converter) (max 14bits) – implemented using its analog blocks, PWM- implemented using its digital blocks and a processor core forms such a control loop required for regulation. Other algorithms like cell balancing can be implemented using a firmware logic.
+</p>
+
+#### Secondary Battery - LiPo/Li-ion battery
+
+   -	High Cell Voltage (3.7V under Load)
+   -	Lightweight, High power Density
+   -	Output capability better than other (like- LEAD TYPE, NiMH, etc.)
+   -	Higher power to size ratio
+   
+## How to simulate smartwatch in proteus
+  - Install proteus and arduino IDE and all the required libraries.
+  - Make the arrangement of all component according to the schematic.
```

Comment:
Step 2 has to be rewritten. Make it "Download the proteus file from [Design/Proteus](https://github.com/crooknish/Smartwatch/tree/main/Design/Proteus)"